app/graphql/types/user.rb

```diff
@@ -20,5 +20,6 @@ class User < BaseObject
     has_many :personal_access_tokens, Types::PersonalAccessToken
     has_many :ssh_keys, Types::SshKey
     has_many :usergroups, Types::Usergroup
+    has_many :permissions, Types::Permission
```

Comment:
Would it be possible to add this to the user query test?
Not a blocker, tho :)

Reply:
Added to user query test.